HelpSource/Classes/Complex.schelp

```diff
@@ -1,6 +1,6 @@
 CLASS:: Complex
 summary:: complex number
-categories:: Math
```

Comment:
There will be Quarks that have helpfiles with the `Math` category. Since normally, geometry is a part of math, it would seem better to just drop it (or: and add it as an additional tag with a comma).